debug/src/debug.js

```diff
@@ -75,20 +81,15 @@ export function initDebug() {
 
 		// Check prop-types if available
 		if (typeof vnode.type==='function' && vnode.type.propTypes) {
-			if (vnode.type.displayName === 'Lazy') {
+			if (vnode.type.displayName === 'Lazy' && !vnode.__lazyPropTypesWarned) {
+				vnode.__lazyPropTypesWarned = true;
```

Comment:
can we use a WeakMap/Map for this? it'll avoid a shape change on the vnode.

Reply:
Honestly didn't know that `WeakMap` has such a browser support. As I just tried to do this I saw that we can re-use `warnedComponents` for this :)